README.md

```diff
@@ -79,26 +79,51 @@ To see details of your systems runtime environment (including the location of th
 $ cc-runtime cc-env
 ```
 
-## Debugging
+## Logging
 
-The runtime provides `--log=` and `--log-format=` options. However, it can
-also be configured to log to the system log (syslog or `journald`) such that
+The runtime provides `--log=` and `--log-format=` options. However, you can
+also configure it to log to the system log (syslog or `journald`) such that
 all log data is sent to both the specified logfile and the system log. The
 latter is useful as it is independent of the lifecycle of each container.
 
-To view runtime log output,
+To view runtime log output:
 
 ```bash
 $ sudo journalctl -t cc-runtime
 ```
 
-### Enabling debug for various components
+To view shim log output:
+
+```bash
+$ sudo journalctl -t cc-shim
+```
+
+To view proxy log output:
+
+```bash
+$ sudo journalctl -t cc-proxy
+```
+
+Note:
+
+The proxy log entries also include output from the agent (`cc-agent`) and the
+hypervisor (which includes the guest kernel boot-time messages).
+
+## Debugging
+
+The runtime, the shim (`cc-shim`), the proxy (`cc-proxy`),
+and the hypervisor all have separate `enable_debug=` debug
+options in the [configuration file](#Configuration). By default, all these
```

Comment:
Lines 116-117 suggested rewrite: All of these debug options are disables by default.